src/stealth/stealth.js

```diff
@@ -0,0 +1,60 @@
+import hidingTricks from './hiding_tricks';
+
+/**
+ * Class that handles hiding headless chrome.
+ * The main purpose of this class is to override newPage function and attached selected tricks.
+ */
+class Stealth {
```

Comment:
I think that this Class could be replaced with simple function because we don't need any internal state. What we want is to just execute some code in the context of a newly opened page. And there is a function for that `page.evaluateOnNewDocument(pageFunction[,...args])`